### ISSUE TITLE
SPF & DKIM checking

### ISSUE THREAD

**Issue by thorsheim** [einaros/starttls.info#14]
Include SPF & DKIM record check
1) do they exist?
2) do they look correct?

Protip: there's a Thunderbird plugin to check & verify DKIM.

**Comment by jimfenton** [comment]
It isn't generally possible to check for the existence of a DKIM public key record. The record is stored at {selector}._domainkey.{domain} where {selector} is the key name obtained from the signature. Without a signature, you wouldn't know where to look.

**Comment by thorsheim** [comment]
Of course, my bad Jim. hihi. I just picked it up from a tweet earlier, someone asked for SPF & DKIM.
